src/pages/en/core-concepts/astro-components.md

```diff
@@ -154,25 +154,26 @@ const items = ["Dog", "Cat", "Platypus"];
 
 #### Fragments & Multiple Elements
 
-Remember: an Astro component template can render multiple elements with no need to wrap everything in a single `<div>` or `<>`.
-
-However, when using an Astro JSX-like expression to dynamically create elements, you must wrap these multiple elements inside of a **Fragment** just like you would in JavaScript or JSX. Astro supports using either `<Fragment> </Fragment>` or `<> </>`.
+An Astro component template can render multiple elements with no need to wrap everything in a single `<div>` or `<>`, unlike JavaScript or JSX.
 
 ```astro
 ---
 const items = ["Dog", "Cat", "Platypus"];
 ---
 <ul>
   {items.map((item) => (
-    <>
-      <li>Red {item}</li>
-      <li>Blue {item}</li>
-      <li>Green {item}</li>
-    </>
+    <li>Red {item}</li>
+    <li>Blue {item}</li>
+    <li>Green {item}</li>
   ))}
 </ul>
 ```
 
+However, Astro also supports using either `<Fragment> </Fragment>` or `<> </>`. This may be useful to add [`client:*` directives](/en/reference/directives-reference/) and avoid wrapper elements, as in the following example:
+
+```astro
+<Fragment set:html={htmlString} />
```

Comment:
Not sure if we need this clarifying context here or only in the directives reference, but this might help explain what is going on.

```suggestion
---
const htmlString = '<p>Raw HTML content</p>';
---
<Fragment set:html={htmlString} />
```